src/index/IndexImpl.Text.cpp

```diff
@@ -1087,7 +1087,7 @@ Index::WordEntityPostings IndexImpl::getEntityPostingsForTerm(
 
     // Read the full lists
     Index::WordEntityPostings eBlockWep = readWordEntityCl(tbmd);
-    resultWep = FTSAlgorithms::intersect(matchingContextsWep, eBlockWep);
+    resultWep = FTSAlgorithms::crossIntersect(matchingContextsWep, eBlockWep);
```

Comment:
Do we still need the old `intersect` function for anything,
Or can we simply remove it/ replace it with the new `crossIntersect`?